command/agent/cache/lease_cache.go

```diff
@@ -108,7 +101,7 @@ func NewLeaseCache(conf *LeaseCacheConfig) (*LeaseCache, error) {
 
 	// Create a base context for the lease cache layer
 	baseCtx, baseCancelFunc := context.WithCancel(conf.BaseContext)
-	baseCtxInfo := &ContextInfo{
+	baseCtxInfo := &cachememdb.ContextInfo{
```

Comment:
Can we use `cachememdb.NewContextInfo` here by passing in `conf.BaseContext` as parameter?

Reply:
Fixed in d120680.